### PR TITLE
feat/nats

### PR DESCRIPTION
**Pull request by disturbing**
This change is [Reviewable](https://reviewable.io/reviews/kintohub/charts/4)

### CODE REVIEW

**Comment by disturbing** [comment]
image.repository=bitnami/nats
image.tag=2.1.0-debian-9-r0
replicaCount=1
auth.enabled=true
auth.user=nats_user
auth.password=undefined
auth.token=undefined
clusterAuth.enabled=true
clusterAuth.user=nats_cluster
clusterAuth.password=undefined
clusterAuth.token=undefined
maxConnections=100
maxControlLine=512
maxPayload=65536
writeDeadline=2s

**Comment by bakayolo** [comment]
Missing resources limit

Add values to values.kinto

File: stable/nats/templates/NOTES.txt [hunk 1]
```diff
@@ -1,99 +1,10 @@
...
-    password on "auth.password" parameter.
-
--------------------------------------------------------------------------------
-{{- end }}
-{{- end }}
-{{- end }}
-
 NATS can be accessed via port {{ .Values.client.service.port }} on the following DNS name from within your cluster:
 
    {{ template "nats.fullname" . }}-client.{{ .Release.Namespace }}.svc.cluster.local
```
Comment (bakayolo) [inline stable/nats/templates/NOTES.txt]: Remove `.{{ .Release.Namespace }}.svc.cluster.local`

File: stable/nats/templates/NOTES.txt [hunk 2]
```diff
@@ -1,99 +1,10 @@
...
-
-{{- include "nats.validateValues" . -}}
-
-{{- if and (contains "bitnami/" .Values.image.repository) (not (.Values.image.tag | toString | regexFind "-r\\d+$|sha256:")) }}
-
-WARNING: Rolling tag detected ({{ .Values.image.repository }}:{{ .Values.image.tag }}), please note that it is strongly recommended to avoid using rolling tags in a production environment.
-+info https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/
-
-{{- end }}
+    {{ template "nats.fullname" . }}-monitoring.{{ .Release.Namespace }}.svc.cluster.local
```
Comment (bakayolo) [inline stable/nats/templates/NOTES.txt]: `.{{ .Release.Namespace }}.svc.cluster.local` to be removed

**Review by bakayolo — CHANGES_REQUESTED** [review]

**Pull request merged**